app/lib/common/widgets/error_widget.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+
+class ErrorWidgetTemplate extends StatelessWidget {
```

Comment:
As this is meant to show the empty state - which is a legitimate state - this shouldn't be named "Error"Widget ... It's the NoDataToShow or EmptyState or something...

Reply:
Renaming it to EmptyState